Framework/Kernel/src/InstrumentInfo.cpp

```diff
@@ -284,7 +285,7 @@ void InstrumentInfo::fillLiveData(const Poco::XML::Element *elem) {
   for (unsigned long i = 0; i < connections->length(); ++i) {
     auto *conn = dynamic_cast<Poco::XML::Element *>(connections->item(i));
     try {
-      m_listeners.emplace_back(LiveListenerInfo(conn));
+      m_listeners.emplace_back(LiveListenerInfo(this, conn));
```

Comment:
I think the `emplace_back` call here is not being properly utilised. As in the copy constructor will be called inplace. This should look like:
```cpp
m_listeners.emplace_back(this, conn);
```